docs/tutorials/resultsdb_tutorial.ipynb

```diff
@@ -0,0 +1,887 @@
S5kIIirJymNSkjF1ZAAQUUUEABBVpZwN8krQxqdgoooIACCnRCAYMhnfCguksKKKCAAgq0pUBxZUM+ONKW2zVvBRRQQAEFFFAgCRT/HknTfVZAAQUUUEABBeoTMBhSn4zTFVBAAQUUUKAsASsjymJyIQUUUEABBRRQQAEFFFBAAQUU6ECBbh24bTetgAIKKKCAAgoooIACCiiggAIKKKCAAgoooIACCrS5gMGQNid2AwoooIACCiiggAIKKKCAAgoooIACCiiggAIKKNCRAgZDOlLfbSuggAIKKKCAAgoooIACCiiggAIKKKCAAgoooECbCxgMaXNiN6CAAgoooIACCiiggAIKKKCAAgoooIACCiiggAIdKWAwpCP13bYCVSLw9ttvh2eeeSa8++67LS7xm2++GXiYFFBAAQUUUEABBRRQQAEFFFBAAQXaVmDt2rXh5ZdfDq+88krbbsjcFagCgR5VUMaSRVyyZEm4/vrrC/M22mijcNBBB8X3zOvWrVsYMGBAYb4vqkvgySefDDz69esXjjnmmHUKP2fOnHDfffeFRYsWhQ033DDsu+++YejQoessV+0TGnOob/+WLl0a7rzzzkAQY7fddgtjxoypb9FGp2N8+eWXh8022yxsvPHG8ZgUr9SU7T300ENx9UMPPTQ+L1++PLzzzjth/fXXL87W9woooIACCiiggAIKKKCAAgoooECHCjS3bqZDC53b+A033BBWrFgRRowYETbffPPcnPdfVvs+vr8nlf1q1apVoUeP6qqOr8YyN3QWVG3LkP79+4cDDjggUJFKJe2OO+5Y2M+pU6eGp59+uvDeF00ToPKbiyAV1O2dli1bFq655ppAhfno0aPDvHnz1ikC5eNCPnHixPCxj30sbLXVVuGKK67okPKuU7hWmlCOQ32beu6558KFF14Y1ltvvbBy5coWu8yePTsGnA477LCSAYumbu+QQw4JPFIifwI3JgUUUEABBRRQQAEFFFBAAQUUUKBSBFpSN1Mp+0A5Xn311bDHHnuEyZMnr1Osjt5HWq08++yz4aWXXlqnbJ1tAnXY8+fPr7rdosyUvbOk6gpF5dS7d+8ehg0bFnr27BkrfdNd5VdddVVs+tWnT5/Yrc8nPvGJGHGjtcitt94aXnvttbj8PvvsE0aNGhVznDVrVnjsscdifg888EAYPHhwOPDAA2PrkilTpsS76wm25AMuL7zwQmAeAQPKQauUVIZcMUPKe4sttgj33ntv4EO+884718mrnLKx7WnTpoWtt946jB07Nr+J8PDDD8f3lGX69OmxRQzl33TTTeN0tnn33XeHGTNmhF69eoVx48aFXXbZJe4fC6xZsybcdddd0YvoJMscFqCPuwAAQABJREFUccQR8Zn5DZXvjTfeCJdddlk48sgjC9tjneYmtkWg6/DDDw9Ukj/xxBPrZPXggw+G7bbbLu4HMzlefDDx2XvvvddZvqEJtC7ZaaedQu/evUsuxoed4BpfGu2ZynGorzy4cTwIEvKFV0569NFHox9fgrQi4fzhPODzwGeDOwgIsNCag/M9n5q6vUceeSSuzjlI/py/nKPkn85bos4ESJ5//vn4GZ80aVI8Z1mRAM9FF10U9txzz3D77beHmpqa2PqFz8Utt9wSaDU0fvz4OC0dV85T5nGe8Dndfffdw5ZbbpnfDV8roIACCiiggAIKKKCAAgoooIACBYGW1M0UMsm9aKs6KHoFoc6T+pBBgwaFvfbaK1APSR3PpZdeGrs8p3cd6glTLx2pWE3dx1QnQ/0NdYmLFy+O9ZS8p5ce0ty5c2M9JHUw1L1S75avS6JulvoculDnRl7qobjhOSXq93hQJ0fdDXmn1hTl1semvCrlmW7nubmb+uxqS9S7LViwINan0YNPtaeqbRlSHzxdAm2yySbxQ7///vvHk4zKfirs+dB/+tOfjhXptD7gJCTx4aLfPE7Ij3/84/EueC4StC6h9QkVp/fcc09YuHBhXJ4P/nXXXRcryE8++eT4waYyuVQibyqkuTB9+MMfDlTqkleKeJZTNi4SBADoDqpUV1B0UURLDi4sxx57bNx/ujV67733YpEY64ELEN1NYfLUU08Vts8CXGAI2hA4Yn3KRIsMAk2NlY/1udjxwWiNlLo7Y9v1JSq2ubjnE+/T8clPb+w1611yySUlI5wcO740uLC3dyrHob4ycc4SCCk3EUAjIMFn5+ijj47nTeqCjgADD847zh0CT8WpqdvjfOVBIu9tt902XlDJf4MNNojT+RLn/D3xxBPjZ/Dxxx+P5zgzOSe5CBOE+ehHPxq/VPkS5Ut4hx12iIE0Pl/sV0oEQvh8nHLKKXE/CXCRj0kBBRRQQAEFFFBAAQUUUEABBRQoJdCSuplS+bVFHVSqt+PGYur0CCrceOONMSDBTa7UtfDMjdnUSRanpu5jqpPhhmsCK9QJUe/JTdgkbna96aab4k2qJ5xwQqy/o04mJW5+vfrqq8OECRPC5z//+XijM/WWBEVI1FlSB8RNvtT5EKxJ6zelPjZtrxKeqQNLddCVUJ7mloF9SPV5zc2jEtbrdMGQ4cOHxzENqLSlQphKeu4uJ3JFczDGEeHDzzzueE+pb9++sbUGlbG0GuHDRkUtAQg+oCNHjgwzZ86Mi3PgyZfumbiQ7brrrjFKmfIqfubudLZNhTLLbrPNNvGDzXLllI0LxVFHHRUrcUsFQ8iHABAtPtJFjHKlrsLYHoEQ9oX9pvKZJmgpEQhhXcy4a579Ti0KGisf5Tn11FPj9lN+bf1MC5gUEU7b4n1zBvfGggh1cUAkBUIw+8AHPpA2U1HPBNhef/31wqO5lfsEQjDgPCdgSBdWBBMIOnE+8ODzw7lDi6vWTCl/Pn8pf44vX6p0yzVw4MAY2CTIQdAun2hZwvlH2Tl3+eLkc0r/l7QcejkbHCylt956K95NQH58buleLd2xkJbxWQEFFFBAAQUUUEABBRRQQAEFFGgrgbaog6KlBDeTUp
```

Comment:
You don't need the intermediate object since `load()` is a class method. So, the second line could just read:
```python
results = DbExperimentData.load("6cc66a03-de43-4d36-9c39-f56d5ee8c011", provider.service("experiment"))
```

Reply:
Ugh, this is showing up strangely (hard to give local feedback on a jupyter notebook).

I think it is also good to show how to get the service from the provider, since presumably if one is loading an ExperimentData into a fresh notebook kernel, you might not have access to an existing `ExperimentData` to grab it from.

Reply:
Good point.